### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/con-pca-api.git con-pca-api
    cd con-pca-api
    git remote add skeleton https://github.com/cisagov/skeleton-docker.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/dependabot.yml README.md 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -17,17 +16,22 @@ updates:
...
-      - dependency-name: docker/setup-buildx-action
-      - dependency-name: docker/setup-qemu-action
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/github-script
+      # - dependency-name: actions/upload-artifact
+      # - dependency-name: docker/build-push-action
+      # - dependency-name: docker/login-action
+      # - dependency-name: docker/setup-buildx-action
+      # - dependency-name: docker/setup-qemu-action
+      # - dependency-name: github/codeql-action
```
Comment (jsf9k) [inline .github/dependabot.yml]: There Dependabot ignore directives should be uncommented.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Con-PCA API #
 
-[![GitHub Build Status](https://github.com/cisagov/con-pca-api/workflows/build/badge.svg)](https://github.com/cisagov/con-pca-api/actions/workflows/build.yml)
-[![CodeQL](https://github.com/cisagov/con-pca-api/workflows/CodeQL/badge.svg)](https://github.com/cisagov/con-pca-api/actions/workflows/codeql-analysis.yml)
-[![Known Vulnerabilities](https://snyk.io/test/github/cisagov/con-pca-api/badge.svg)](https://snyk.io/test/github/cisagov/con-pca-api)
+[![GitHub Build Status](https://github.com/cisagov/skeleton-docker/workflows/build/badge.svg)](https://github.com/cisagov/skeleton-docker/actions/workflows/build.yml)
+[![CodeQL](https://github.com/cisagov/skeleton-docker/workflows/CodeQL/badge.svg)](https://github.com/cisagov/skeleton-docker/actions/workflows/codeql-analysis.yml)
+[![Known Vulnerabilities](https://snyk.io/test/github/cisagov/skeleton-docker/badge.svg)](https://snyk.io/test/github/cisagov/skeleton-docker)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/skeleton-docker.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/skeleton-docker/context:python)
```
Comment (jsf9k) [inline README.md]: Please correct these URLs to point to `cisagov/con-pca-api` instead of `cisagov/skeleton-docker`.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I've requested a few simple fixes.

File: .github/dependabot.yml (hunk 1 above)
Comment (itsmostafa) [reply .github/dependabot.yml]: @jsf9k resolved with [2471778](https://github.com/cisagov/con-pca-api/pull/860/commits/2471778c39ce2cf5c920faf13098476b7f361212)

File: README.md (hunk 2 above)
Comment (itsmostafa) [reply README.md]: @jsf9k resolved with [e87220f](https://github.com/cisagov/con-pca-api/pull/860/commits/e87220f2c52667b8738b039a47608285c99fc713)

**Review by itsmostafa — APPROVED** [review]

**Review by jsf9k — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 [![GitHub Build Status](https://github.com/cisagov/con-pca-api/workflows/build/badge.svg)](https://github.com/cisagov/con-pca-api/actions/workflows/build.yml)
 [![CodeQL](https://github.com/cisagov/con-pca-api/workflows/CodeQL/badge.svg)](https://github.com/cisagov/con-pca-api/actions/workflows/codeql-analysis.yml)
 [![Known Vulnerabilities](https://snyk.io/test/github/cisagov/con-pca-api/badge.svg)](https://snyk.io/test/github/cisagov/con-pca-api)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/con-pca-api.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/con-pca-api/context:python)
```
Comment (dav3r) [inline README.md]: This should be removed now that we have phased out LGTM:
```suggestion

```

**Review by dav3r — CHANGES_REQUESTED** [review]
I noticed one small thing that was overlooked.

File: README.md (hunk 3 above)
Comment (itsmostafa) [reply README.md]: @dav3r resolved with [cb8e78c](https://github.com/cisagov/con-pca-api/pull/860/commits/cb8e78c251df016e790a6080624e869a4d34d50e)

Comment (dav3r) [reply README.md]: @itsmostafa - Looks like you accidentally deleted the blank line after the LGTM badge in https://github.com/cisagov/con-pca-api/pull/860/commits/cb8e78c251df016e790a6080624e869a4d34d50e.  Can you please correct that?

Comment (itsmostafa) [reply README.md]: @dav3r done! [3729c4b](https://github.com/cisagov/con-pca-api/pull/860/commits/3729c4b386ae887241410862c42bdd511be44ace)

**Pull request merged**